# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libgd/fontconfig.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Needed for modular X.org.

  - Rene Rebe <rene@exactcode.de>

--- gd-2.0.33/Makefile.vanilla	2005-10-16 12:30:08.000000000 +0200
+++ gd-2.0.33/Makefile	2005-10-16 12:30:37.000000000 +0200
@@ -228,7 +228,7 @@
 CCDEPMODE = depmode=gcc3
 CFLAGS = -g -O2
 CPP = cpp
-CPPFLAGS = -I/usr/include/freetype2 
+CPPFLAGS = -I/usr/include/freetype2 -I/usr/X11/include
 CYGPATH_W = echo
 DEFS = -DHAVE_CONFIG_H
 DEPDIR = .deps
